topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -336,6 +336,52 @@ user interface, then translate the user's selections back into the command to be
 on writing a wrapper. Although this section will cover the full process and many of the options available to
 anyone wrapping a tool, a more complete list is available in [the Galaxy tool docs](https://docs.galaxyproject.org/en/latest/dev/schema.html).
 
+## Installing Planemo
+
+Planemo is an important tool within the Galaxy development workflow. Although it will be covered in greater detail in the
+next section of this tutorial, one of its functions is necessary for the tool wrapper development. `planemo` is available
+via the python package manager `pip` and `conda`. If you not have conda installed we suggest to use `pip` for installing
+`planemo`. To that end, before beginning this section, please install planemo locally by running
+
+> ### {% icon hands_on %} Hands-on: Installing `planemo` via pip
+>
+> It is advised to install `planemo` in a virtual environment (step 1-2).
+>
+> 1. Create a virtual environment: `virtualenv -p python3 ~/.venv/`. Here `~/.venv/` is the path where the virtual environment should be crated and you may adapt the path to your needs. With `-p python3` we make sure that a `python` intepreter version 3 is used.
```

Comment:
Do we need to tell folks how to install virtualenv? (Or `venv`, whichever is easier)? Or should we have them use conda since presumably they've installed conda?

Reply:
I thought about installing `planemo` with `conda`. I remember that I asked on a gitter channel and got the advice to opt for python venv.

Reply:
It used to be a horrible mess if you started galaxy and conda was on path. I wouldn't go down that route now.

Reply:
Ahh that's too bad. That would've made installation easier for users.